microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md

```diff
@@ -1,24 +1,24 @@
 ---
-description: Get started with Remote Debugging Windows 10 devices
-title: Get started with remote debugging Windows 10 Devices
+description: Get started with Remote Debugging Windows 10 and later devices
+title: Get started with remote debugging Windows 10 and later Devices
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.date: 04/23/2021
 ms.topic: article
 ms.prod: microsoft-edge
-keywords: microsoft edge, web development, f12 tools, devtools, remote, debugging, windows 10, windows, device portal
+keywords: microsoft edge, web development, f12 tools, devtools, remote, debugging, windows 10, Windows 10 and later, Windows 11, windows, device portal
 ---
-# Get started with remote debugging Windows 10 Devices
+# Get started with remote debugging Windows 10 and later Devices
 
-Remote debug live content on a Windows 10 device from your Windows or macOS computer.  This tutorial teaches you the following tasks.
+Remote debug live content on a Windows 10 and later device from your Windows or macOS computer.  This tutorial teaches you the following tasks.
 
-*   Set up your Windows 10 device for remote debugging and connect to it from your development machine.
-*   Inspect and debug live content on your Windows 10 device from your development machine.
-*   Screencast content from your Windows 10 device onto a DevTools instance on your development machine.
+*   Set up your Windows 10 and later device for remote debugging and connect to it from your development machine.
+*   Inspect and debug live content on your Windows 10 and later device from your development machine.
+*   Screencast content from your Windows 10 and later device onto a DevTools instance on your development machine.
```

Comment:
```suggestion
*   Set up your Windows device for remote debugging and connect to it from your development machine.
*   Inspect and debug live content on your Windows device from your development machine.
*   Screencast content from your Windows device onto a DevTools instance on your development machine.
```
The lead-in sentence for this list specifies "10 and later", not needed again in list, and is distracting when repeated too often.